hbase-server/src/main/java/org/apache/hadoop/hbase/mob/MobFileCleanerChore.java

```diff
@@ -92,10 +92,6 @@ private void checkObsoleteConfigurations() {
     }
   }
 
-  public MobFileCleanerChore() {
```

Comment:
I'm pretty sure there are a few more usages you need to clean up: https://github.com/search?q=repo%3Aapache%2Fhbase%20%22new%20MobFileCleanerChore%22&type=code